src/agents/gc/OSConfig_Proto.schema.mof

```diff
@@ -0,0 +1,71 @@
+#pragma include ("cim_schema_2.26.0.mof")
+#pragma include ("MSFT_Qualifiers.mof")
+#pragma include ("OMI_BaseResource.mof")
+#pragma include ("MSFT_Credential.mof")
+
+[ClassVersion("1.0.0")]
+class OSConfig_PrototypeResource : OMI_BaseResource
+{
+  [key] 
+  string PrototypeClassKey;
+
+  [write, ValueMap{"Present","Absent"}, Values{"Present","Absent"}]
+  string Ensure;
+  
+  [read, Description ("Reported string type value" )]
+  string ReportedString;

```

Comment:
>ReportedString [](http://example.com/codeflow?start=9&length=14)

Marius i see set of two kind of variables here. reported* and expected*. what is the reason to have these pairs?

Reply:
Yes. This is a PoC and marked explicitly reported vs desired settings to be exchanged between the NRP and OSConfig.